src/styles/blocks/cards.scss

```
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+.main--card {
```

```suggestion
.main__card {
```
And move it to the file with main Block